datasets/ONS-UK-Economic-Accounts-balance-of-payments-current-account/main.py

```diff
@@ -1,36 +1,16 @@
 # ---
 # jupyter:
 #   jupytext:
-#     formats: ipynb,py
 #     text_representation:
 #       extension: .py
 #       format_name: light
 #       format_version: '1.5'
-#       jupytext_version: 1.3.3
+#       jupytext_version: 1.6.0
 #   kernelspec:
 #     display_name: Python 3
 #     language: python
 #     name: python3
 # ---
 
-from gssutils import *
-from gssutils.metadata import THEME
-import pandas as pd
-
-# +
-# %%capture
-
-# %run "B1.py"
-
-# %run "B2_B2A_B3_B3A.py"
-
-# %run "B4_B4A_B4B.py"
-
-# %run "B5_B5A.py"
-
-# %run "B6_B6A.py"
-
-# %run "B6B_B6B2_B6B3_B6C_B6C2_B6C3.py"
-
-# %run "B7_B7A.py"
-
+for scripts = ['B1.py', 'B2_B2A_B3_B3A.py', 'B4_B4A_B4B.py', 'B5_B5A.py', 'B6_B6A.py', 'B6B_B6B2_B6B3_B6C_B6C2_B6C3.py', 'B7_B7A.py']: 
+    %run $script
```

Comment:
This doesn't actually run for me, I got the following error. 
```python
SyntaxError: invalid syntax
```

I used this instead and it worked for me: 
```python
import glob

py_files = [i for i in glob.glob('*.{}'.format('py'))]

for i in py_files:
    file = "'" + i + "'"
    if file.startswith("'main") == True:
        continue
    %run $file
```

Reply:
Thanks Shannon for reviewing the pull request. The feedback is noted and changes will be made shortly.